src/commands/index.ts

```diff
@@ -29,10 +29,6 @@ export function load(client: any): any {
   client.appdistribution.group.list = loadCommand("appdistribution-group-list");
```

Comment:
I assume keeping `client.appdistribution.groups` and removing `client.appdistribution.group` makes more sense.

..and probably the commands themselves could/should be renamed: `appdistribution-group-*` to `appdistribution-groups-`.

Reply:
Renamed the files, and updated this file to make the commands available under both the old and new names when using firebase-tools as a module.

Reply:
Thanks!